src/meta/ActiveHostsMan.h

```diff
@@ -32,97 +33,48 @@ struct HostInfo {
     }
 
     int64_t lastHBTimeInSec_ = 0;
-};
-
-class ActiveHostsMan;
-
-class ActiveHostsManHolder final {
-public:
-    ActiveHostsManHolder() = delete;
-    ~ActiveHostsManHolder() = delete;
-
-    static ActiveHostsMan* hostsMan() {
-       static auto hostsMan
-           = std::make_unique<ActiveHostsMan>(FLAGS_expired_hosts_check_interval_sec,
-                                              FLAGS_expired_threshold_sec);
-       return hostsMan.get();
-    }
+    bool isOnline_ = false;
```

Comment:
Do we need this field?  If exists in ActiveHostMan, it means the host is online.

Reply:
Maybe it is not mandatory, but necessary, so we can get active hosts from memory instead of get in kvstore.